rca/static_src/sass/abstracts/_variables.scss

```diff
@@ -40,18 +40,75 @@ $color--meta-lightbg: $color--dark-grey;
 
 $color--focus: $color--blue;
 
+// --------------------------------- Font faces -------------------------------------
+
+/* source-sans-3-latin-400-normal */
+@font-face {
+    font-family: 'Source Sans 3';
+    font-style: normal;
+    font-display: swap;
```

Comment:
I wonder if `block` would be more appropriate to prevent FOUT? 

I accept the argument that `swap` displays text on the page more quickly on first load, but it can be jarring as the type shifts once the desired font is loaded in.